gemfiles/rails_6.0_sidekiq_6.2.gemfile

```diff
@@ -0,0 +1,9 @@
+# This file was generated by Appraisal
+
+source "https://rubygems.org"
+
+gem "activejob", "~> 6.0.0"
+gem "activesupport", "~> 6.0.0"
+gem "sidekiq", "~> 6.1.0"
```

Comment:
This one says "6.1.0"?

Reply:
Thanks Tim. I noticed the typeo in the Appraisals file but forgot to reinstall them.

Reply:
Just force pushed a fix.